lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -4,7 +4,7 @@
 
 <% unless calculator.country_without_consular_facilities? %>
   <% if calculator.ceremony_country == 'vietnam' %>
-    <%= render partial: 'consular_fees_in_cash_visa_master.govspeak.erb',
+    <%= render partial: 'link_to_consular_fees_for_vietnam.govspeak.erb',
```

Comment:
The "consular_fees_in_cash_visa_master" partial has been renamed in PR #2592. It's possible that'll be merged before this PR (they're both out for fact check) so I'm not going to change anything in this PR just yet.